src/applications/vaos/services/slot/index.js

```diff
@@ -37,24 +35,15 @@ export async function getSlots({
   clinicId,
   startDate,
   endDate,
-  convertToUtc = false,
+  convertToUtc,
 }) {
   try {
-    const parseStartDate = parseISO(startDate);
-    const parseEndDate = parseISO(endDate);
-
-    const start = convertToUtc
-      ? parseStartDate.toISOString()
-      : format(parseStartDate, "yyyy-MM-dd'T'HH:mm:ssxxx");
-    const end = convertToUtc
-      ? parseEndDate.toISOString()
-      : format(parseEndDate, "yyyy-MM-dd'T'HH:mm:ssxxx");
-
```

Comment:
Moved to 'getAvailableV2Slots' since formatting concerns should be where they are used.